transip/service/dns.py

```diff
@@ -0,0 +1,50 @@
+"""
+DNS related objects
```

Comment:
P.S. You can run
```
pylint --rcfile=.pylintrc transip
```
to test the configuration locally.

P.P.S. I will add this to the docs :-)